src/webgpu/util/conversion.ts

```diff
@@ -1112,6 +1112,35 @@ export function scalarTypeOf(ty: Type): ScalarType {
   throw new Error(`unhandled type ${ty}`);
 }
 
+/** @returns the concrete type of the given Type.
```

Comment:
put the `@returns` on another ` * ` prefixed line - I don't think it'll treat the `@returns` as a keyword otherwise.

Reply:
Done.